integration/firestore/package.json

```diff
@@ -6,9 +6,8 @@
     "build:deps": "lerna run --scope @firebase/'{app,firestore}' --include-dependencies build",
     "build:persistence": "INCLUDE_FIRESTORE_PERSISTENCE=true gulp compile-tests",
     "build:memory": "INCLUDE_FIRESTORE_PERSISTENCE=false gulp compile-tests",
+    "karma:singlerun": "karma start --single-run",
     "prettier": "prettier --write '*.js' '*.ts'",
-    "test": "yarn build:memory; karma start --single-run; yarn build:persistence; karma start --single-run;",
```

Comment:
I think we can maybe keep these or introduce "test:memory" and "test:persistence".

Reply:
Lines 12 and 14 define exactly "test:memory" and "test:persistence"!

Reply:
I just wanted to clean up unused entries in this "scripts" list to reduce future confusion.